ui/app/styles/_layout.scss

```diff
@@ -51,23 +50,53 @@ aside {
   padding-top: scale.$lg-6;
 }
 
+.grid {
```

Comment:
The use of CSS `grid` is awesome. But since we may eventually have multiple grid setups in the future, the name of this class feels very "generic". Could we consider something like `.grid-menu-with-panel` (or other) to be more descriptive?